include/gdstk/utils.hpp

```diff
@@ -231,6 +232,28 @@ inline uint64_t hash(const char* key) {
 extern FILE* error_logger;
 void set_error_logger(FILE* log);
 
+#pragma pack(push, 1)
+typedef struct {
```

Comment:
Are there any publicly available references to this that we could link in a comment?

Reply:
This structure was provided to me by the raith team, it wont be online anywhere